tutorial_boosting.md

```diff
@@ -62,15 +66,40 @@ Run it in a terminal with the command:
 python search_with_boosting.py
 ```
 
-Now you can check ```results[]```, the products that correspond to the boost condition became reranked.
+Now you can check results[]. The products corresponding to the boost condition became reranked.
+
+## Some notes about boosting
 
-Please note that setting boost strength to 1.0 gives the item a strong promotion. However, it does not necessarily mean that the boosted item will be the top result at all times, nor that other items will be excluded. 
-Results could still be shown even when none of them matches the condition. Also, results that are significantly more relevant to the search query can still trump your heavily favored but irrelevant items.
+Please note that setting the boost strength to 1.0 gives the item a strong promotion. However, it **does not necessarily mean that the boosted item will be the top result at all times**, nor that other items will be excluded. 
+Results could still be shown even when none of them matches the condition. 
+
+Also, results that are **significantly more relevant** to the search query can still trump your heavily favored but irrelevant items.
 
 You can combine up to 10 boost specifications in one search request. In this way you may apply really sophisticated boosting rules to your search request.
 
+## Try different boosting conditions
+
+Feel free to test products boosting yourself, right here in Cloud Shell environment.
+
+Just find the comment: 
+
+"# TRY DIFFERENT BOOST CONDITIONS HERE:"
+
+and replace the condition expression for something like this:
```

Comment:
_It's better to use replace with smth. instead of replace for._

and replace the condition expression with something like this: